docs/whats_new.rst

```diff
@@ -5,255 +5,282 @@ What's New
-        ):
+  - FLARE API now has the same set of APIs as the Admin Client.
+  - Allows users to use almost all of the commands from python API or notebooks.
 
-Prior to release 2.3.0,
+- Docker Support
 
-Wait_time_after_min_received: this means after min_response received, we will wait wait_time_after_min_received.
+  - NVFLARE cloud CSP startup scripts now support deployment with docker containers in addition to VM deployment.
+  - provision command now supports detached docker run, in addition to the interactive docker run.
 
-In Release 2.3.0: 
+- Flare Dashboard
 
-Wait_time_after_min_received: If min_response received, but not all responses are received, we will wait wait_time_after_min_received.
-If all responses are received, there is no wait.
+  - Prior to the 2.4.0, the Flare dashboard can only run within a docker container.
+  - In the 2.4.0, the Flare dashboard can now run locally without docker for development.
 
-5. Behavior changes to POC ``–stop``
-====================================
-In 2.2.x version, the POC stop will try to kill the process directly regardless the system state. 
+- Run Model Evaluation Without Training
 
-In 2.3.0 version, the stop command will try with the following:
+  - In the 2.4.0 release, users can now run cross-validation without having to re-run the training.
+  - `Enable re-run cross-validation without training workflow (WIP) <https://github.com/NVIDIA/NVFlare/pull/2035>`_.
 
-  #. Connect to the server
-  #. If server can be connected, then list active jobs
-  #. Abort all active jobs
-  #. Call system shutdown, and wait for system to gradually shutdown
-  #. Wait for system to shut down with max_timeout of 30 seconds
-  #. After that, we try kill the process (this was the entirety of the 2.2.x behavior)
+- Communication Enhancements
 
-6. Scatter and Gather Controller API changes
-============================================
-A new argument has been added to :class:`ScatterAndGather<nvflare.app_common.workflows.scatter_and_gather.ScatterAndGather>`. ``allow_empty_global_weights`` is
-an optional boolean to determine whether or not to allow empty global weights and defaults to False.
+  - We added the application layer ping between Client Job process and Server parent process to replace the gRPC timeout.
+    Previously, we noticed if the gRPC timeout is set too long, the cloud provider (eg. Azure Cloud) will kill the connection after 4 minutes.
+    If the timeout setup is too short (such as 2 mins), the underlying gRPC will report too many pings.
+    The application level ping will avoid both issues to make sure the server/client is aware of the status of the processes.
+  - gRPC Asynchronous communication vs synchronous communication: we noticed that in an unstable network environment, async communication may result in job crashes,
+    while sync communication results in more stable communication. We have changed the default settings to sync communication from asynchronous communication.
```

Comment:
Ideally, we should set the default to AIO version for the Server side, and non-AIO version for the client side.
@nvidianz and @yhwen , can we do this?  The challenge here is how do we know the process is server or client at the time of loading the drivers.

Reply:
Will keep note of this for future. Resolving for this pr